### PR TITLE
story/TYTA-129/global-overview-page-for-user

### PR DESCRIPTION
**Pull request by remirubis**
### Description

Implementation of basic overview page, I'm waiting backend implementation of time task lead by @FabienHP.

### CODE REVIEW

File: src/Router.jsx [hunk 1]
```diff
@@ -29,13 +30,15 @@ const Router = () => {
...
+      element: (
+        <RequireAuth>
+          <ConnectedLayout />
+        </RequireAuth>
+      ),
       children: [
-        { path: '', element: <ListTeams /> },
+        { path: 'overview', element: <Overview /> },
+        { path: 'teams', element: <ListTeams /> },
         { path: 'user/:teamID', element: <AddUserToTeam /> },
```
Comment (kilianpichard) [inline src/Router.jsx]: Hum, maybe we should keep the same pattern from routes as backend ? So /team/:teamId/user ?

File: src/components/Layouts/Connected/Connected.layout.jsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ const Main = styled('main', { shouldForwardProp: (prop) => prop !== 'open' })(
 );
 
 const ConnectedLayout = () => {
-  const [open, setOpen] = React.useState(false);
+  const [open, setOpen] = React.useState(true);
```
Comment (kilianpichard) [inline src/components/Layouts/Connected/Connected.layout.jsx:36]: Nice change, wanted to do it on my PR but forgot

**Review by kilianpichard — COMMENTED** [review]
Some comments

File: src/Router.jsx (hunk 1 above)
Comment (remirubis) [reply src/Router.jsx]: Yes i agree with your feedback! We can set `/team/:teamID` for AddUserToTeam no?

Comment (kilianpichard) [reply src/Router.jsx]: Hum.. maybe we should keep the /team/:teamID to a possible view to display all details of the team, and we have the update route also so maybe /user at the end for add user and /update for the update ?

File: src/Router.jsx [hunk 3]
```diff
@@ -29,14 +30,16 @@ const Router = () => {
...
+        <RequireAuth>
+          <ConnectedLayout />
+        </RequireAuth>
+      ),
       children: [
-        { path: '', element: <ListTeams /> },
-        { path: 'user/:teamID', element: <AddUserToTeam /> },
+        { path: 'overview', element: <Overview /> },
+        { path: 'teams', element: <ListTeams /> },
+        { path: 'user/:teamID/users', element: <AddUserToTeam /> },
```
Comment (kilianpichard) [inline src/Router.jsx]:
```suggestion
        { path: 'teams/:teamID/users', element: <AddUserToTeam /> },
```

**Review by kilianpichard — COMMENTED** [review]
Comment on route

**Pull request merged**